lightning/src/ln/channelmanager.rs

```diff
@@ -3843,6 +3863,33 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> MessageSend
 				result = NotifyOption::DoPersist;
 			}
 
+			if self.startup_complete.load(Ordering::Relaxed) != 0 {
+				let new_feerate = self.fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::Normal);
```

Comment:
Hmmmm should we use `ConfirmationTarget::HighPriority` by default ?

Are we operating under the assumptions that a) our counterparty can go offline at anytime and b) mempool can spikes making the obsolete the pre-signed feerate to pass mempools min feerate ?

If we want to avoid feerate overhead in case of unilateral closure, and the counterparty is online, we can still `update_fee` adjust-in-time before to enforce the closing?

Reply:
I suppose if we broadcasted our commitment transaction quickly after our counterparty went offline (which we won't but assuming we did), then we could use Normal and its ok. I think we should use `Normal` until we have anchor, then switch anchor to `Normal` (or maybe `Background * 2`?) then switch non-anchor to something higher? The issue is really that `HighPriority` may just be a very high fee and users would complain.

Reply:
Well this feerate should only play out in case of non-cooperative closure, which should be the minority of all channel closures so intuitively I would say we can be generous with it and user shouldn't complain ?
Though i concede you can still have a lot of closures for liquidity operations/channel management tools, and you would like to be conservative....

W.r.t to anchor support, i think we could extend our `FeeEstimator` API to ask for mempool min feerate ? And I can extend Core's `estimaterawfee` to return this value, pending for package relay deployment ? And yes switch legacy fee-bumping to a default higher feerate.

Reply:
> W.r.t to anchor support, i think we could extend our FeeEstimator API to ask for mempool min feerate ?

Yea, this is a good question. I'm not sure what the right answer is, sadly most fee estimators have no concept of "mempool min fee".